fuel_utility.py

```diff
@@ -56,6 +56,9 @@ def getByCategory(self, category):
     def getByModel(self, model_name):
         self.appendSingleModel(model_name)
     
+    def getModelUrl(self, mode_name)
+        return FUEL_URI+model_name+"/2/"+model_name+".zip" 
```

Comment:
can we extract model version too?

Reply:
[nit] do we need space between + ? 
not sure about our coding guideline.

Reply:
I am actually not sure if the "2" stands for a model version and how that url gets updated..in that case, I don't think we should explicitly name it as constant having certain version significance in this script